.github/workflows/manage-pending-labels-closed.yml

```
@@ -0,0 +1,17 @@
+# When a new comment is added to an issue, if it had the Stale or Awaiting User Response labels,
+# then those labels will be removed, providing it was not user khulnasoft who added the commend. 
+name: 🎯 Remove Pending Labels on Close
+on:
+  issues:
+    types: [closed]
+jobs:
+  remove-labels:
+   runs-on: ubuntu-latest
+   steps:
+   - name: Remove Labels when Closed
+     uses: actions-cool/issues-helper@v2
```

actionlint

the runner of "actions-cool/issues-helper@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)